plugins/inputs/jolokia/jolokia.go

```diff
@@ -132,37 +226,37 @@ func (j *Jolokia) Gather(acc telegraf.Accumulator) error {
 		tags["port"] = server.Port
```

Comment:
@saiello I should have caught this in the original plugin, but here you are overwriting the telegraf-wide `host` tag, could you change these tags:

- server
- port
- host

to:

- server_name
- server_port
- server_host